save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/TestResourcesSelection.kt

```diff
@@ -201,6 +212,33 @@ fun testResourcesSelection(
                         setSelectedLanguageForStandardTests
                     )) {}
 
+                    div("input-group-prepend") {
+                        input(type = InputType.text, name = "itemText") {
+                            key = "itemText"
+                            attrs.set("class", "form-control")
+                            attrs {
+                                value = props.execCmd
+                                placeholder = "Execution command:"
+                                onChangeFunction = {
+                                    setExecCmd(it)
+                                }
+                            }
+                        }
+                    }
```

Comment:
This can be extracted into a function taking value, placeholder and onChange as parameters

Reply:
Good advise, fixed